<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="2" failed="0" total="2" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="58" started-at="2022-01-10T07:07:17Z" finished-at="2022-01-10T07:07:17Z">
    <groups>
    </groups>
    <test name="Test3" duration-ms="57" started-at="2022-01-10T07:07:17Z" finished-at="2022-01-10T07:07:17Z">
      <class name="EndtoEndFrameWork.validateNavigationBar">
        <test-method status="FAIL" signature="intialize()[pri:0, instance:EndtoEndFrameWork.validateNavigationBar@663c9e7a]" name="intialize" is-config="true" duration-ms="0" started-at="2022-01-10T12:37:17Z" finished-at="2022-01-10T12:37:17Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at resources.Base.initializeDriver(Base.java:39)
	at EndtoEndFrameWork.validateNavigationBar.intialize(validateNavigationBar.java:27)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:216)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:143)
	at org.testng.TestRunner.beforeRun(TestRunner.java:631)
	at org.testng.TestRunner.run(TestRunner.java:599)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:368)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:402)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- intialize -->
        <test-method status="SKIP" signature="ValidateNavigationBar()[pri:0, instance:EndtoEndFrameWork.validateNavigationBar@663c9e7a]" name="ValidateNavigationBar" duration-ms="9" started-at="2022-01-10T12:37:17Z" finished-at="2022-01-10T12:37:17Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ValidateNavigationBar -->
        <test-method status="SKIP" signature="tearDown()[pri:0, instance:EndtoEndFrameWork.validateNavigationBar@663c9e7a]" name="tearDown" is-config="true" duration-ms="0" started-at="2022-01-10T12:37:17Z" finished-at="2022-01-10T12:37:17Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- EndtoEndFrameWork.validateNavigationBar -->
    </test> <!-- Test3 -->
    <test name="Test2" duration-ms="58" started-at="2022-01-10T07:07:17Z" finished-at="2022-01-10T07:07:17Z">
      <class name="EndtoEndFrameWork.validateTitleTest">
        <test-method status="FAIL" signature="intialize()[pri:0, instance:EndtoEndFrameWork.validateTitleTest@1ca3b418]" name="intialize" is-config="true" duration-ms="0" started-at="2022-01-10T12:37:17Z" finished-at="2022-01-10T12:37:17Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at resources.Base.initializeDriver(Base.java:39)
	at EndtoEndFrameWork.validateTitleTest.intialize(validateTitleTest.java:25)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:216)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:143)
	at org.testng.TestRunner.beforeRun(TestRunner.java:631)
	at org.testng.TestRunner.run(TestRunner.java:599)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:368)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:402)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- intialize -->
        <test-method status="SKIP" signature="ValidateTitle()[pri:0, instance:EndtoEndFrameWork.validateTitleTest@1ca3b418]" name="ValidateTitle" duration-ms="13" started-at="2022-01-10T12:37:17Z" finished-at="2022-01-10T12:37:17Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ValidateTitle -->
        <test-method status="SKIP" signature="tearDown()[pri:0, instance:EndtoEndFrameWork.validateTitleTest@1ca3b418]" name="tearDown" is-config="true" duration-ms="0" started-at="2022-01-10T12:37:17Z" finished-at="2022-01-10T12:37:17Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- EndtoEndFrameWork.validateTitleTest -->
    </test> <!-- Test2 -->
  </suite> <!-- Suite -->
</testng-results>
